platform/core/src/types/index.ts

```diff
@@ -0,0 +1,5 @@
+import type * as HangingProtocol from './HangingProtocol'
+
+export {
```

Comment:
Why isn't this one export type { ...
It looks like it is otherwise the same as the import.